docs/enhancements/boot-status-reporter.md

```diff
@@ -0,0 +1,129 @@
+---
+title: boot-status-reporter
+authors:
+  - "@nmagnezi"
+creation-date: 2022-09-22
+last-updated: 2022-09-22
+---
+
+# Assisted Boot Reporter
+
+## Summary
+
+Create a new reporting service that runs as early as possible when hosts are booted from disk to collect additional 
+information and send it to `assisted-service`.
+
+## Motivation
+
+There are multiple occurrences of clusters that fail to install at the stage where
+hosts manage to pull ignition but fail to get to a point where the `assisted-controller`
+pod runs and calls home to `assisted-service` with logs.
+
+With that missing information, there's not a whole lot we can do to understand what
```

Comment:
I think this enhancement should discuss the boot-reporter functionality in multi nodes cluster installation and SNO installation. 
The node stages updates and the logs gathering we have in place are different (lacking in case of SNO).